GCEWindowsAgent/main.go

```diff
@@ -78,34 +80,46 @@ func logStatus(name string, disabled bool) {
 }
 
 func parseConfig(file string) (*ini.File, error) {
+	empty, _ := ini.InsensitiveLoad([]byte{})
 	d, err := ioutil.ReadFile(file)
 	if err != nil {
-		return nil, err
+		return empty, err
 	}
-	return ini.InsensitiveLoad(d)
+	cfg, err := ini.InsensitiveLoad(d)
+	if err != nil {
+		return empty, err
+	}
+	return cfg, nil
 }
 
 func runUpdate() {
-	cfg, err := parseConfig(configPath)
-	if err != nil && !os.IsNotExist(err) {
-		logger.Errorf(err.Error())
-	}
-	if cfg == nil {
-		cfg, _ = ini.InsensitiveLoad([]byte{})
+	cfgPath := lxConfigPath
+	if runtime.GOOS == "windows" {
+		cfgPath = winConfigPath
 	}
 
-	config = cfg
+	var err error
+	config, err = parseConfig(cfgPath)
+	if err != nil && !os.IsNotExist(err) {
+		logger.Errorf("error parsing config %s: %s", cfgPath, err.Error())
+	}
 
 	var wg sync.WaitGroup
-	for _, mgr := range []manager{newWsfcManager(), &addressMgr{}, &accountsMgr{}, &diagnosticsMgr{}} {
+	var mgrs []manager
+	if runtime.GOOS == "windows" {
+		mgrs = []manager{newWsfcManager(), &addressMgr{}, &accountsMgr{}, &diagnosticsMgr{}}
+	} else {
+		mgrs = []manager{&addressMgr{}}
+	}
+	for _, mgr := range mgrs {
 		wg.Add(1)
 		go func(mgr manager) {
 			defer wg.Done()
 			if mgr.disabled() || (!mgr.timeout() && !mgr.diff()) {
 				return
 			}
 			if err := mgr.set(); err != nil {
-				logger.Errorf(err.Error())
+				logger.Errorf("error running %v manager: %s", mgr, err.Error())
```

Comment:
Here and elsewhere:
Capitalize Log entries and use %v for error, and you can just use err not err.Error() with the %v format call